src/white-bread.md

```diff
@@ -0,0 +1,38 @@
+4. Add (42g) softened butter, 1 tbsp at a time. Once everything has been incorporated, knead in the machine for 30 seconds more.
+5. Take dough out of bowl and lightly shape into a ball. Lightly grease a bowl with some vegetable or olive oil (or any flavorless oil). Place dough in greased bowl and cover with greased plastic wrap. Let rise at room temperature for 1 to 1.5 hrs or until double in size. *if you used instant yeast, your dough might need less time. Be sure to check your dough periodically. Once you see it's doubled in size, no need to wait the full 1 - 1.5 hrs to do the next step.*
+6. Lightly grease a 9 x 4 ½ inch loaf pan.
+7. Deflate dough and place on lightly floured surface. Using a rolling pin, roll out your dough to half an inch thick and wide enough to fit in your loaf pan.
+8. Tightly roll your dough into a log and pinch the seam. Place inside the loaf pan (seam side down), cover with a damp towel, greased plastic wrap, shower cap, or an inverted loaf pan (if you have a cake carrier dome that works too), and let rise at room temperature for 45 mins to 1 hr or until double in size. *same note as step 5. Your dough might need less time than this. Once it has doubled in size, go to the next step to prevent your loaf from collapsing during baking. If you forgot to check your dough and it has risen more than one inch above the rim of your loaf pan,  it might be over-proofed which could lead to your loaf collapsing / deflating in the oven. You can test this by poking the side of the loaf with your finger. If the dent fills out halfway after a few seconds, your dough is properly proofed and ready for the oven. If the dent disappears right away, dough in under-proofed and you need to wait a few more minutes. If the dent does not disappear, it means the dough is over-proofed. To save over-proofed dough, punch it down to release all the gas, re-shape into a log and then let it proof for an additional 20 mins (each time you proof the dough will need less time) or until it has doubled in size or risen one inch above the rim of your pan.*
+9. Preheat oven to 350ºF. *Depending on how fast your oven heats up, you can do this during the last 10 - 15 mins of your second proof. This way your dough won't be in danger of over-proofing while you wait for your oven to finish pre-heating*
+10. Place loaf pan in the oven and bake  at 350ºF for 35 to 40 mins or until golden brown. Take loaf out of pan and place on a cooling rack. You can test if the loaf is properly cooked by tapping the bottom. If it sounds hollow, it's done. If there's no sound, the inside might be undercooked. Return the loaf in the oven for another 5 to 10 mins (monitor closely to prevent burning). Let your loaf cool completely before slicing.
```

Comment:
I like all the detail in this recipe! Bread is a hard thing to make right...

Reply:
I straight up copied this recipe from a comment on a YouTube video